noxfile.py

```diff
@@ -133,15 +133,17 @@ def system_emulated(session):
     subprocess.call(["gcloud", "components", "install", "beta", "bigtable"])
 
     hostport = "localhost:8789"
+    session.env["BIGTABLE_EMULATOR_HOST"] = hostport
+
     p = subprocess.Popen(
         ["gcloud", "beta", "emulators", "bigtable", "start", "--host-port", hostport]
     )
 
-    session.env["BIGTABLE_EMULATOR_HOST"] = hostport
-    system(session)
-
-    # Stop Emulator
-    os.killpg(os.getpgid(p.pid), signal.SIGTERM)
+    try:
+        system(session)
+    finally:
+        # Stop Emulator
+        os.killpg(os.getpgid(p.pid), signal.SIGKILL)
```

Comment:
Ensure that no matter how the tests complete, we really kill the emulator. `SIGTERM` leaves the process running on my local machine, so use `SIGKILL`.